src/Apps/Partner/Components/PartnerArtists/PartnerArtistsCarousel/PartnerArtistsCarousel.tsx

```diff
@@ -27,25 +27,27 @@ export const PartnerArtistsCarousel: React.FC<PartnerArtistsCarouselProps> = ({
   ) {
     return null
   }
-
-  const artists = extractNodes(partner.artistsConnection)
+  const { slug } = partner
+  const artists = compact(partner.artistsConnection.edges)
 
   return (
     <Shelf alignItems="flex-start">
-      {artists.map(artist => (
-        <CellArtistFragmentContainer
-          key={artist.internalID}
-          artist={artist}
-          to={`/partner/${partner.slug}/artists/${artist.slug}`}
-          FollowButton={
-            <FollowArtistButtonQueryRenderer
-              id={artist.internalID}
-              contextModule={ContextModule.recommendedArtistsRail}
-              size="small"
-            />
-          }
-        />
-      ))}
+      {artists
+        .filter(artist => artist !== null)
+        .map(artist => (
+          <CellPartnerArtistFragmentContainer
+            key={artist.node?.internalID}
+            artistPartnerEdge={artist}
+            to={`/partner/${slug}/artists/${artist.node?.slug}`}
+            FollowButton={
+              <FollowArtistButtonQueryRenderer
+                id={artist.node?.internalID!}
+                contextModule={ContextModule.recommendedArtistsRail}
```

Comment:
I don't think this needs the `recommendedArtistsRail` context since that's not what this is. We don't necessarily need a contextModule; in which case you can just leave this as the default by not passing in a `FollowButton`

Reply:
ah cool done as well!